pages/_app.tsx

```diff
@@ -115,25 +116,27 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
       >
         <DefaultSeoWithLocale />
         <ThemeProvider disableSmoothScrolling={isSearchOpen ? true : disableSmoothScrolling} headComponent={Head}>
-          <div
-            sx={{
-              position: 'absolute',
-              left: 0,
-              right: 0,
-              top: 0,
-              minHeight: '768px',
-              backgroundImage: `url('${process.env.BASE_PATH}/img/page-background.png')`,
-              backgroundSize: 'cover',
-              backgroundPosition: 'center top',
-              '@media (min-width: 1440px)': {
```

Comment:
We want to keep the media query for `aspectRatio`, so that on viewports smaller than 1440px, this `div` (in which the background image is centered) is 100% of the viewport's width (due to the `left: 0` and `right: 0`) and exactly 768px tall (due to the `minHeight`). But on viewports >= 1440px, the `div` keeps growing in height to maintain the `1440/768` aspect ratio, which is the same as the background image.

Reply:
I removed it because there was an issue with the horizontal scroll and this is what it was causing it. But seems like it is fine now and it's not longer happening with media query. so I am adding it back, thank you!

Reply:
The issue with the horizontal scroll is fixed by the `overflow: hidden` parent you added. :)